cookbook/configuration/environments.rst

```diff
@@ -6,7 +6,7 @@ How to Master and Create new Environments
 
 Every application is the combination of code and a set of configuration that
 dictates how that code should function. The configuration may define the
-database being used, whether or not something should be cached, or how verbose
+database being used, if something should be cached or how verbose
```

Comment:
Are you sure that this comma is useless? The text is describing three separate things, so I find it normal that they are separated by the comma and the `or`.